widgets/cpu_openbsd.lua

```diff
@@ -0,0 +1,60 @@
+-- cpu_openbsd.lua - provide CPU usage on OpenBSD
+-- Copyright (C) 2019  Enric Morales
+--
+-- This file is part of Vicious.
+--
+-- Vicious is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as
+-- published by the Free Software Foundation, either version 2 of the
+-- License, or (at your option) any later version.
+--
+-- Vicious is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
```

Comment:
Big OOF to me: I copied the header from my other project and it has the wrong license name :grin:  While I fixed all of that in the copyright branch, I missed it while reviewing your notmuch widget type too.

Reply:
Heh! I thought it was a new GNU license d'oh! I guess removing Affero from line 14 and 16 does the trick, doesn't it?